scopesim/effects/electronic.py

```diff
@@ -204,6 +205,10 @@ def apply_to(self, obj, **kwargs):
             if exptime is None:
                 exptime = from_currsys("!OBS.dit", self.cmds) * \
                           from_currsys("!OBS.ndit", self.cmds)
+                logger.debug("Exptime auto-determined from DIT and NDIT.")
```

Comment:
I think that the AutoExposure should not do anything in the case of the user explicitly specifying a DIT and NDIT. So I'd say we just `!OBS.autoexpset`to False here and return. And then below we can just set `!OBS.autoexpset` to True.

Reply:
I was wondering for some time why this was there at all. Might there be a reason?

> And then below we can just set `!OBS.autoexpset` to True.

Yeah that's actually what I did at first, assuming it wouldn't reach that part when DIT, NDIT are given...